config/config/common-config.yaml

```diff
@@ -134,6 +139,13 @@ falco:
       # Can be skipped if eBPF JIT is enabled from outside the container
       hostNetwork: false
 
+    # override the URL used for downloading driver modules, e.g. to use a self hosted file server in an air-gapped environment
+    repoURL: ""
```

Comment:
Is this only used for the module driver? In that case I'd prefer if it was under the key `module`.